app/assets/javascripts/discourse/app/controllers/preferences/tracking.js

```diff
@@ -0,0 +1,126 @@
+import Controller from "@ember/controller";
+import discourseComputed from "discourse-common/utils/decorators";
+import { NotificationLevels } from "discourse/lib/notification-levels";
+import I18n from "I18n";
+import { or } from "@ember/object/computed";
+import { popupAjaxError } from "discourse/lib/ajax-error";
+
+export default Controller.extend({
```

Comment:
We can start writing controllers using native JS class. There is an example of how to do it in https://github.com/discourse/discourse/blob/4244b1c57d40bda3b07d1b7de8fcdb24ebd800ad/app/assets/javascripts/discourse/app/controllers/preferences/sidebar.js.